pages/booklet.tsx

```diff
@@ -1,39 +1,45 @@
 import Head from "next/head"
 import Link from "next/link"
-import React from "react"
+import React, {useEffect} from "react"
```

Comment:
oh i see you added useEffect here, i think using this with this code as reference could help!
https://github.com/wliu080/japan-prayer-guide/blob/03fc197933fd5b235f49101aabdbe375ddc0839a/components/toggleHeader.tsx#L45C3-L67C66

Reply:
I think the `react-responsiveness` is cleaner, but we could probably discuss this after the conference.  This `useEffect` was left from my testing with a previous version so I need to remove it